.github/workflows/install.yml

```
@@ -205,60 +209,197 @@ jobs:
+  resource_check:
+    name: Check Resources
     needs: meta
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.10)

[error] the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)